webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -433,13 +433,15 @@ def update_corr_index(
         ctx = dash.callback_context
         triggered_id = ctx.triggered[0]["prop_id"].split(".")[0]
 
-        if triggered_id == parent.uuid("parameter-selection-store-resp"):
+        if triggered_id == parent.uuid("parameter-selection-store-resp") or (
+            triggered_id == "" and not click_data
```

Comment:
Which is the case `triggered_id == ""`? I could not figure it out.

Reply:
ya, very good, I thought yesterday "this probably deserves an inline comment, or an explanation in the commit message" : D

that case is you have selected ensembles, and responses, and you refresh the page.
at the moment, the `corr_xindex` store is not of session type - it is not remembered across reloads (I asked Dan about this in an issue, but didn't get a response, he must have overlooked it, and then i forgot until seeing it again yesterday), but ensemble and response selection is remembered.
So you reload, ensembles are chosen, responses are chosen, but if we do not have that check `triggered_id == "" and not click_data`, which is basically the callback firing for the first time on page load, then the `corr_xindex` will not get populated, and you end up without an index.

does that make sense?

if so, what do we do on the team / what would you prefer or suggest: an inline comment, or a descriptive git commit message?

Reply:
ah i see i actually put it in the commit message of commit 9d3823d26f86dcbd217cdbd721853c69bdf56fdc :sweat_smile: but maybe it is not very clear?

Reply:
It makes a lot of sense 🎉 
I would suggest to have it documented as code, e.g. having a variable `page_reloaded = (triggered_id == "" and not click_data)`, and then having `if triggered_id == parent.uuid("parameter-selection-store-resp") or page_reloaded`.
I think we are not in the state to trust that we will get much information from a git commit message (I think you are the only one in the squad keeping this good habit so far, we should definitely do it more), and an inline comment would be as ugly as this alternative I suggested, but I prefer to have the programatic comment ;-)  What do you think? Maybe the name `page_reloaded` could be of a better choice?

Reply:
ah, that's a great solution, thank you ^^
i guess the name could be `first_invocation` ? technically it's not only on reload, i suppose...

Reply:
used `first_invocation` now.